src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -379,7 +379,6 @@ internal string CreateSearchQuery(string[] fields, string SearchTerm, bool IsPac
                     {
                         //Means that the first term is a category when we will be using the FullCategoryName for making a specific search based in the category
                         trimmedSearchTerm = matchingCategory?.FullCategoryName;
-                        occurQuery = Occur.MUST;
```

Comment:
Reason behind removing this?

Reply:
when searching for criteria like "file path" with MUST we were creating a query like this one (consider that the word 'file' is a Category):

`SELECT .... FROM ..... WHERE CATEGORY = 'file' AND (NAME LIKE 'path' or NAME LIKE ''path*)`

so it was returning the two nodes in which Category = 'Core.File' that match the criteria related with the Name (the Name should contain 'path'), 
But the user is expecting the "file path" node - this one **didn't match** due that belongs to the Category = "Core.Input" so removing the MUST makes the query more flexible showing also the nodes that matches the Name = 'file path' **OR** the Category = "Core.File".
Please let me know if this is clear enough.